src/plots/cartesian/axes.js

```diff
@@ -202,16 +202,18 @@ axes.saveRangeInitial = function(gd, overwrite) {
     var axList = axes.list(gd, '', true),
         hasOneAxisChanged = false;
 
-    var ax, isNew, hasChanged;
-
     for(var i = 0; i < axList.length; i++) {
-        ax = axList[i];
+        var ax = axList[i];
```

Comment:
I'm always happy to see these kinds of small style changed. No more scrolling all the way to the top of a file to see whether some variable had a value previously!